<?xml version="1.0" encoding="UTF-8"?>
<testsuites tests="3" failures="1" disabled="0" errors="0" timestamp="2016-08-11T02:33:36" time="0.001" name="AllTests">
  <testsuite name="firstTest" tests="1" failures="0" disabled="0" errors="0" time="0">
    <testcase name="abs" status="run" time="0" classname="firstTest" />
  </testsuite>
  <testsuite name="secondTest" tests="1" failures="0" disabled="0" errors="0" time="0.001">
    <testcase name="abs" status="run" time="0" classname="secondTest" />
  </testsuite>
  <testsuite name="third" tests="1" failures="1" disabled="0" errors="0" time="0">
    <testcase name="abs" status="run" time="0" classname="third">
      <failure message="mytest.cpp:16&#x0A;Value of: abs( -1 )&#x0A;  Actual: 1&#x0A;Expected: 2" type=""><![CDATA[mytest.cpp:16
Value of: abs( -1 )
  Actual: 1
Expected: 2]]></failure>
    </testcase>
  </testsuite>
</testsuites>
